types/sensative/strip/meta.json

```diff
@@ -1,19 +1,15 @@
 {
-  "name": "All Strips (LoRaWAN)",
+  "name": "Strips Multi-sensor +Guard",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed